internal/policy_source/repo.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package policy_source
```

Comment:
Why not "policy/source"?

Reply:
Just noting that we replaced one line of go-getter with 150 lines of code here, not trying to offend, but by this objective measure this is not a good decision.

Reply:
No offense taken, but more accurately stated, imo, there's an equivalent number of lines of code to checkout the source, but the inclusion of the "repo" structure makes it look like more. If I'm honest, I am not sure that the "repo" model here is worth keeping if it's the only source we're drawing from.